Modules/Core/Controllers/SettingsController.php

```
@@ -74,12 +127,12 @@ public function index(Request $request): \Illuminate\Http\RedirectResponse|\Illu
             return redirect()->route('settings.index');
         }
         // Load required resources using Eloquent
-        $invoice_groups  = InvoiceGroup::query()->all();
-        $tax_rates       = TaxRate::query()->all();
-        $email_templates = EmailTemplate::query()->all();
-        $payment_methods = PaymentMethod::query()->all();
-        $templates       = Template::query()->all();
-        $custom_fields   = \Modules\CustomFields\Models\CustomField::query()->all();
+        $invoice_groups  = InvoiceGroup::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\Modules\Invoices\app\Models\InvoiceGroup' in method 'index'. (undefined)

(StaticAccess)